### PR TITLE
feat(stories): adds @storybook/addon-docs

### PR DESCRIPTION
**Pull request by sharvit**
## Updated Packages

* [ ] root
* [ ] @theforeman/builder
* [ ] @theforeman/env
* [ ] @theforeman/eslint-plugin-foreman
* [x] @theforeman/stories
* [ ] @theforeman/vendor
* [ ] @theforeman/vendor-dev
* [ ] @theforeman/vendor-core

## PR Type

* [ ] Bugfix
* [x] Feature
* [ ] Code style update (whitespace, formatting, missing semicolons, etc.)
* [ ] Refactoring (no functional changes, no api changes)
* [ ] Build related changes
* [ ] CI related changes
* [ ] Documentation content changes
* [ ] Other… Please describe:

## Description

Adding [@storybook/addon-docs](https://github.com/storybookjs/storybook/tree/next/addons/docs)

1. Adds a `DocsPage` for each story, see: https://github.com/storybookjs/storybook/blob/next/addons/docs/docs/docspage.md
2. Support writing stories using `.stories.mdx` files, see: https://github.com/storybookjs/storybook/blob/next/addons/docs/docs/mdx.md

## Does this PR introduce a breaking change?

* [ ] Yes
* [ ] No

## Does this PR fixes open issues?

* [ ] Yes
* [x] No

## Do you use this PR in another repository?

* [ ] Yes
* [ ] No

### CODE REVIEW

File: packages/stories/README.md [hunk 1]
```diff
@@ -87,6 +182,21 @@ Options:
 
 ## Available storybook addons
 
+### @storybook/addon-docs
```
Comment (amirfefer) [inline packages/stories/README.md]: looks awesome !

File: packages/stories/package.json [hunk 2]
```diff
@@ -40,8 +42,10 @@
     "@theforeman/builder": "^3.10.1",
     "@theforeman/vendor": "^3.10.1",
     "babel-loader": "^8.0.6",
+    "babel-plugin-react-docgen": "^3.2.0",
     "commander": "^4.0.1",
     "node-sass": "^4.5.0",
+    "react-is": "^16.12.0",
```
Comment (amirfefer) [inline packages/stories/package.json]: I don't see any use for it, is it needed?

Comment (sharvit) [reply packages/stories/package.json]: Yes, see: https://github.com/storybookjs/storybook/blob/c0f3f71e0307d27d72bd142b9f9ad8413a675731/addons/docs/react/README.md#mdx

**Pull request merged**